documentation/23-mariposa/README.md

```diff
@@ -0,0 +1,152 @@
+Title: Mariposa integration
+----
+Text:
+
+There are a number of cases where we've wanted for lifecycle hooks to be able to
```

Comment:
These docs get published to https://www.joyent.com/containerpilot and the publishing platform we use there is unfriendly to manually-wrapped text. So no need for line breaks here.